### DISCUSSION TITLE
remove Tx mode off?

### DISCUSSION THREAD

**Discussion by proddy** [emsesp/EMS-ESP32#342] [Ideas]
I'm thinking of removing the 'off' option in Tx mode. Without Tx active non of the version checks would work and its pretty useless unless you only want to sniff the EMS traffic.

For those panicking about EMS-ESP writing back to the EMS and it's devices there's the new 'read only' setting.

@MichaelDvP  ?

**Comment by mvjt** [comment]
I´m fine with EMS-ESP probing the devices, but I would like to avoid writing back and changing parameters. So I would be fine with "Read-only".

**Comment by MichaelDvP** [comment]
Remove the selection from web-settings, but leave the tx-mode. If someone like to sniff he has to use the console, set tx-mode off there and sniff the telegrams.

**Comment by proddy** [comment]
done
